lib/linguist/languages.yml

```diff
@@ -1244,6 +1244,16 @@ Emacs Lisp:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 102
+Email:
```

Comment:
Wouldn't EML be a better name?

Reply:
"Email" is more human readable and won't be confused/conflict with other potential languages (https://github.com/NCEAS/eml, http://xml.coverpages.org/eml.html), but it may not matter.